keras_cv/models/object_detection/yolo_v8/yolo_v8_backbone.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2023 The KerasCV Authors
```

Comment:
You mentioned this works out of the box with CSP Darknet.

can we remove this special case one and achieve comparable numerics with our training script?  What augmentations are missing?  

If we're missing any, or you need help constructing them please file an issue against me and I'll looking into it

Reply:
Haifeng is working on porting the CSPDarkNet to the backbone API, and then we should be able to swap them out.

In terms of missing augmentations, it's just `Mosaic` -- although maybe that's already ready?

Reply:
"comparable numerics" here referred to MAP scores on a reference dataset when training from scratch.

I have already verified the forward-pass numerics for the end-to-end model and the backbone.